pvlib/test/test_modelchain.py

```diff
@@ -263,21 +263,16 @@ def constant_spectral_loss(mc):
     mc.spectral_modifier = 0.9
 
 @requires_scipy
-@pytest.mark.parametrize('spectral_model, expected', [
-    ('sapm', [182.338436597, -2.00000000e-02]),
-    pytest.mark.xfail(raises=NotImplementedError)
-    (('first_solar', [179.371460714, -2.00000000e-02])),
-    ('no_loss', [181.604438144, -2.00000000e-02]),
-    (constant_spectral_loss, [163.061569511, -2e-2])
-])
-def test_spectral_models(system, location, spectral_model, expected):
+@pytest.mark.parametrize('spectral_model', ['sapm', 'first_solar', 'no_loss'])
+
+def test_spectral_models(system, location, spectral_model):
+    times = pd.date_range('20160101 1200-0700', periods=3, freq='6H')
+    weather = pd.DataFrame(data=[0.3, 0.5, 1.0], index=times, columns=['precipitable_water'])
     mc = ModelChain(system, location, dc_model='sapm',
                     aoi_model='no_loss', spectral_model=spectral_model)
-    times = pd.date_range('20160101 1200-0700', periods=2, freq='6H')
-    ac = mc.run_model(times).ac
-
-    expected = pd.Series(np.array(expected), index=times)
-    assert_series_equal(ac, expected, check_less_precise=2)
+    spectral_modifier = mc.run_model(times=times, weather=weather).spectral_modifier
+    print(spectral_modifier)
```

Comment:
I try to avoid putting print functions in the final tests. In case you are not already aware of it, the pytest ``--pdb`` option can be useful for avoiding print in development e.g. ``pytest pvlib/test/test_modelchain.py::test_spectral_models --pdb``

Reply:
Removed, I had that print to debug the test.